docs/sources/administration/roles-and-permissions/access-control/_index.md

```diff
@@ -14,6 +14,8 @@ weight: 120
 
 # Role-based access control (RBAC)
 
+> Note: Available only in Grafana Enterprise.
```

Comment:
Some thoughts:

I think it's available for Enterprise and Cloud Advanced? Or am I wrong?

I feel like we should have a disclaimer like this for every RBAC page, as users will likely navigate to them directly rather than through the index page.

We should probably also link to Enterprise and Cloud Advanced, something like what we do [here](https://github.com/grafana/grafana/blob/main/docs/sources/enterprise/query-caching.md?plain=1#L20).

Reply:
I agree, users might happen upon any of these pages without seeing the _index page. And we should add Cloud Advanced, if it's available with that.

Reply:
We don't list RBAC as a feature for Grafana Cloud. If anything, it's explicitly called out as an "Enterprise Stack Self-managed" feature (under the old feature name "fine-grained access control") on the Cloud pricing page: https://grafana.com/pricing/

Reply:
Assuming RBAC is not available, in part or full, in any Cloud edition:

```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}).
```

Avoiding the use of "only" since not all features included or associated with RBAC are Enterprise-only, and linking to Enterprise from the notice. This is also parallel in style to all other such notices added as part of the administration docs refactoring.

Reply:
RBAC is available in Grafana Enterprise and Grafana Cloud Advanced, so the pricing page is wrong. Have let @divygoelgrafana know.